install/installer/pkg/components/server/constants.go

```diff
@@ -17,6 +17,8 @@ const (
 	chargebeeMountPath                     = "/chargebee"
 	stripeSecretMountPath                  = "/stripe-secret"
 	githubAppCertSecret                    = "github-app-cert-secret"
+	IAMSessionPort                         = common.ServerIAMSessionPort
+	IAMSessionPortName                     = "iam-session"
```

Comment:
Do we want to couple the port name to a particular component? Yes, today IAM is the only component using it but I think we could make it more general to not create that name-coupling. I think `session` works

Reply:
ok

Reply:
done